support-frontend/stories/landingPage/Countdown.stories.tsx

```diff
@@ -34,81 +34,90 @@ function Template(args: CountdownProps) {
 	);
 }
 
+function dummy(b: boolean){
```

Comment:
can this be removed?

Reply:
I see further down it's being passed in so maybe it needs  refactored so it uses the countdown?

Reply:
I've removed the dummy function, thanks